README.md

```diff
@@ -35,12 +35,31 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
 ### CI Deployment
 1. Create another cdk project and depend on this package
 2. Import the config / ci stacks alongside the other resources
-   ```
+   ```typescript
    new CIConfigStack(app, 'CI-Config-Beta', {});
-   new CIStack(app, 'CI-Beta', ciSettings);
+   new CIStack(app, 'CI-Beta', ciSettings, {});
    ```
-3. Update the `ciSettings` according to the environment needs such as SSL or strict deployment, see [CIStackProps](./lib/ci-stack.ts) for details.
-4. Deploy using the CI system of your choice.
+3. Import `DeployAwsAssets` stack to deploy aws assets as per your needs.
+4. We currently support deploying public ECR and is deployed as follows -
+   ```typescript
+    new DeployAwsAssets(app, `OpenSearch-CI-Deploy-Assets-${defaultEnv}`, {
+      // This will delete the ECR repository once the stack is destroyed. 
+      // Default removal policy (if not specified) is RemovalPolicy.DESTROY
+      removalPolicy: RemovalPolicy.DESTROY,
+      // account where the main node is deployed
+      mainNodeAccountNumber: ciStack.account,
+      envName: 'dev',
+      env: {
+          // public ECR repositories can only be created in us-east-1
+          // https://github.com/aws/aws-cli/issues/5917#issuecomment-775564831
+          region: 'us-east-1',
+      },
+      repositories: [
+        'opensearch',
+      ],
+    }); ```
```

Comment:
Convert them into words instead of copying the codebase

Reply:
The point describe how to deploy the assets as in previous points and comments describe the properties. Is there something specific you are looking for?

Reply:
We already have this code in our codebase, you can just point at it and explain what you are trying to say

Reply:
The reason I added this here in the `readme` as it is done for [here](https://github.com/opensearch-project/opensearch-ci/blob/main/README.md?plain=1#L39) is because defines every property that deploys the stack which should be a part of the readme point which talks about deploying `DeployAwsAssets` stack. The parameters in `ci-stack.ts` can then change on how user wants to configure it.

Reply:
Feels weird to see the same code base in readme. Customer can go and see for themselves. 
Can you rethink and rephrase this? 
Thanks!

Reply:
Updated the readme